homeassistant/components/hive/climate.py

```diff
@@ -145,6 +143,11 @@ def preset_modes(self):
         """Return a list of available preset modes."""
         return SUPPORT_PRESET
 
+    async def async_added_to_hass(self):
+        """When entity is added to Home Assistant."""
+        await super().async_added_to_hass()
```

Comment:
Is this needed?

Reply:
I wanted to be thorough, didn't have time to check if it really was.